### ISSUE TITLE
Roadmap for registry:2.7.2 and beyond

### ISSUE THREAD

**Issue by brandonmcclure** [distribution/distribution#3311]
[registry:2.7.1 was released on January 17 2019](https://github.com/docker/distribution/releases/tag/v2.7.1) there seems to have been a more recent image published on [docker hub](https://hub.docker.com/layers/registry/library/registry/2.7.1/images/sha256-e09ed8c6c837d366a501f15dcb47939bbbb6242bf3886270834e2a0fa1555234?context=explore) though.

The ROADMAP.md was last updated 5 years ago.

I can see commits happening, but issues like #3189 that have not had any response have me wondering what the future of this project is. As someone who is new to docker this is concerning to me, as having a reliable/robust registry infrastructure seems to be central to implementing any sort of container workload in any sort of production capacity.

**Comment by rajivml** [comment]
@stevvooe @dmcgowan @BrianBland @aaronlehmann I see that you guys are top committers for this project, can anyone of you please clarify about the sustainability of this project i.e. can we take dependency on this project or not to setup docker private registry for hosting images onprem

**Comment by wy65701436** [comment]
The v2.7.2/v2.8 is planned, but we don't have a ETA about it. Hopefully we can have a 2.7.2 soon.

**Comment by dirkmueller** [comment]
we had already commits, is there any way the current state of things can be released as a new version? it would be kinda nice to have a new release of things at least every 6 months.

**Comment by xvzf** [comment]
Guys is there an update on it

**Comment by milosgajdos** [comment]
We will do `2.8` release soon -- once this PR is merged (one people are back from holidays) https://github.com/distribution/distribution/pull/3552 -- and then push the `2.8` image to Hub, etc.; from then on we will only consider high sev security fixes. but it will be the last `2.x` release. All the future effort will go towards releasing `v3` hopefully in Q1 in 2022

We are working on getting the "edge" (`main` branch) merges to Hub, soon, too. See this PR: https://github.com/distribution/distribution/pull/3315; once that's in, we'll push every `main` change into Hub under `distribution` namespace (https://hub.docker.com/repository/docker/distribution/distribution).

**Comment by DavidSpek** [comment]
Closing this issue as v2.8.2 is currently the latest release and this doesn’t seem relevant anymore.
